projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-component.scss

```diff
@@ -558,6 +566,11 @@
         @extend %igx-grid__hierarchical-expander !optional;
     }
 
+    @include e(hierarchical-expander, $m: empty) {
```

Comment:
Is this modifier class additive? I.e. are you adding it to the class list, or do you replace igx-grid__hierarchical-expander with igx-grid__hierarchical-expander--empty?

Reply:
In the current implementation `igx-grid__hierarchical-expander--empty` replaces the default `igx-grid__hierarchical-expander` class.